### ISSUE TITLE
label 관련 질문 드립니다

### ISSUE THREAD

**Issue by gyjung94** [homink/deepspeech.pytorch.ko#5]
안녕하세요
default label이 영어 알파뱃으로 되어있던데, 제가 label_ko.json으로 바꾸어 돌리는 것이 맞을까요?

**Comment by gyjung94** [comment]
그리고 학습 진행에 따라 CER은 항상 0이 나오고, WER은 너무 큰 수가 나오는데, 어디서 문제가 생긴 것인지 예측이 어렵습니다. 혹시 가능하시면 도움 부탁드립니다

**Comment by gyjung94** [comment]
그리고 실례가 아니라면 혹시 학습이 끝난 한국어 pre-trained 모델을 공유해 주시기는 어려울까요?
